README.md

```diff
@@ -11,9 +11,11 @@ Installation and Contribution
 ==============================
 See the project website  [`MADlib Home`](http://madlib.apache.org/) for links to the
 latest binary and source packages. For installation and contribution guides,
-and other useful information
+and other useful information such as compiling from source
```

Comment:
Compiling from source is actually in https://cwiki.apache.org/confluence/display/MADLIB/Installation+Guide#InstallationGuide-CompileFromSourceCompilingFromSource.
We need to navigate through multiple links to reach here from the main MADlib wiki page. Can we use this link directly for `compiling from source on Linux` specifically?

Reply:
I put the top level link only since the pages and headings change in the wiki and wanted to avoid broken link.  But I will put the sub-link, no harm.